_node/how-to-guides/deployment/configuring-custom-ports.md

```diff
@@ -14,6 +14,8 @@ permalink: /:collection/:path:output_ext
 
 Node for Cloud 66 is designed to be a simple, hands-off solution for deploying Node applications. As such the complexity around exposing and using ports is largely automated and not editable.
 
+By default your Cloud 66 application will be set to listen on port `3000`, so please ensure your application code is configured accordingly.
```

Comment:
Not actually sure what the default port is, if you're sure it's 3000 that's fine.

Reply:
It's 3000. A magic goat told me this. THEREFORE IT MUST BE TRUE. Also Vic mentioned it.